hbp_knowledge/tau/ryan2019.bel

```diff
@@ -327,108 +405,136 @@ p(HGNC:MAPT,pmod(CONSO:hyperphosphorylation)) -> a(CONSO:"Tau oligomers")
 a(CONSO:"Tau oligomers") -> a(GO:"neurofibrillary tangle")
 UNSET MeSHAnatomy
 UNSET MeSHDisease
+UNSET Confidence
 
 SET Evidence = "It was later found
 that tau phosphorylation was inversely regulated by OGlcNAc
 addition in vitro, in tissue culture cells, and ex vivo
 in rat brain slices (Figure 4)"
+SET Confidence = "High"
 SET Species = "10116"
 SET MeSHAnatomy = "Brain"
 p(RGD:Mapt,pmod(CONSO:"O-GlcNAcylation")) neg p(RGD:Mapt,pmod(Ph))
 UNSET MeSHAnatomy
 UNSET Species
+UNSET Confidence
 
 SET Evidence = "Furthermore, downregulation of OGT leads to increased tau
 phosphorylation and onset of AD pathology."
+SET Confidence = "High"
 act(p(HGNC:OGT)) -| p(HGNC:MAPT,pmod(Ph))
 act(p(HGNC:OGT)) neg path(MESH:"Alzheimer Disease")
+UNSET Confidence
 
 SET Evidence = "Twelve-month old triple-tg mice (3xTg-Ad) displayed reduced levels of tau
 O-GlcNAcylation consistent with tau hyperphosphorylation in the hippocampus."
 # triple-tg mice (3xTg-Ad)
+SET Confidence = "Low"
 SET Species = "10090"
 SET MeSHAnatomy = "Hippocampus"
-p(MGI:Mapt,pmod(CONSO:"O-GlcNAcylation")) neg p(MGI:Mapt,pmod(CONSO:hyperphosphorylation))
+p(MGI:Mapt,pmod(CONSO:hyperphosphorylation)) neg p(MGI:Mapt,pmod(CONSO:"O-GlcNAcylation"))
 UNSET MeSHAnatomy
 UNSET Species
+UNSET Confidence
```

Comment:
@sgebel @cthoyt please check this statement.

Reply:
okay